src/liballoc/boxed.rs

```diff
@@ -126,7 +126,8 @@ impl<T: ?Sized> Box<T> {
         Box(Unique::new_unchecked(raw))
     }
 
-    /// Consumes the `Box`, returning the wrapped raw pointer.
+    /// Consumes the `Box`, returning the wrapped raw pointer, which
+    /// will be properly aligned and non-null.
```

Comment:
So, `rustdoc` uses the first line as a summary, and this ends up being a bit long. Could you maybe change it to

```
/// Consumes the `Box<T>`, returning a raw pointer.
///
/// The pointer will be properly aligned and non-null.
```

Reply:
Thanks @steveklabnik, I didn't know that.